chrome/content/zotero/zoteroPane.js

```diff
@@ -1346,7 +1347,7 @@ var ZoteroPane = new function()
 			}
 		}
 		
-		yield ZoteroItemPane.blurOpenField();
+		yield this.itemPane._itemDetails.blurOpenField();
```

Comment:
We shouldn't access a private property from outside the file. The point of that convention is to know that any changes are limited to the file itself.